app.rb

```diff
@@ -0,0 +1,56 @@
+require './classroom'
+require './rental'
+require './student'
+require './teacher'
+require './book'
+
+class App
+  attr_reader :books, :rental, :people
+
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  def all_book
+    @books&.each do |book|
+      puts "[#{book.class}] - Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def all_person
+    @people&.each do |person|
+      puts "[#{person.class}] - Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def all_rental(id)
+    @people.each do |person|
+      next unless person.id == id
+
+      person.rental.each do |rental|
+        puts "[#{rental.class}] - Book: #{rental.book.title}, Person: #{rental.person.name}, Date: #{rental.date}"
+      end
+    end
+  end
+
+  def create_book(title:, author:)
+    @books << Book.new(title: title, author: author)
+  end
+
+  def create_student(age:, name:, parent_permission: true, classroom: nil)
+    @people << Student.new(age: age, name: name, parent_permission: parent_permission, classroom: classroom)
+  end
```

Comment:
- [ ] Please you app throws an error when I try to create a student, please checkout the screenshot below and kindly fix the issue.

![errorCreatingStudent](https://user-images.githubusercontent.com/65852621/174849437-4f4bfeca-0b5a-44cc-be72-a56797053017.png)